src/webviews/apps/plus/shared/components/integrations-chip.ts

```diff
@@ -377,7 +385,8 @@ export class GlIntegrationsChip extends LitElement {
 						</span>`
 				: html`<span class="integration__content">
 						<span class="integration_details"
-							>GitLens AI features have been disabled by your GitKraken admin</span
+							>GitLens AI features have been
+							disabled${!this.aiSettingEnabled ? ' via settings' : ' by your GitKraken admin'}</span
```

Comment:
Similar to above, would be great to have a button to re-enable.

Reply:
Done